src/data_factory_testing_framework/_expression_runtime/data_factory_expression/data_factory_to_dotnet_evaluator_transformer.py

```diff
@@ -0,0 +1,12 @@
+from typing import Union
+
+from lark import Token, Transformer, Tree, v_args
+
+
+@v_args(tree=True)
+class DataFactoryToDotnetEvaluatorExpressionTransformer(Transformer):
```

Comment:
Should this be the DataFactoryToLogicAppsExpressionTransformer?

Reply:
Renamed to be aligned with our published nuget `Microsoft.Azure.DataFactoryTestingFramework.Expressions`